models/reg_lstm/args.py

```diff
@@ -0,0 +1,31 @@
+import os
+import models.args
+
+
+def get_args():
+    parser = models.args.get_args()
+
+    parser.add_argument('--resume-snapshot', type=str, default=None)
+    parser.add_argument('--bidirectional', action='store_true')
+    parser.add_argument('--bottleneck-layer', action='store_true')
+    parser.add_argument('--single-label', action='store_true')
+    parser.add_argument('--num-layers', type=int, default=2)
+    parser.add_argument('--hidden-dim', type=int, default=256)
+    parser.add_argument('--mode', type=str, default='static', choices=['rand', 'static', 'non-static'])
+    parser.add_argument('--dataset', type=str, default='Reuters', choices=['SST-1', 'SST-2', 'Reuters', 'AAPD', 'IMDB', 'Yelp2014'])
+    parser.add_argument('--words-dim', type=int, default=300)
```

Comment:
`--word-dim`?

Reply:
Hmm, it's consistently words-dim everywhere. I should change it in every model or I would be breaking some of them.